lua/swamp/misc/sh_titles.lua

```diff
@@ -213,3 +276,25 @@ AddTitle("", {
 --NOMINIFY
 --TODO: titles where you don't have a description but the title itself hints at what you do? like a secret achievement but not totally secret
 --TODO: if threshold=true, its 1 but you can put better text on the button in that case
+
+if SERVER then
+    local TitleFunctions =
+    {
+        ["weapon_gauntlet"] = function(atk) atk:AddStat("snaps") end,
+        ["weapon_slitter"] = function(atk,vic) if vic:IsPlayer() && not vic:IsAfk() then atk:AddStat("knifekill") end end,
+        ["dodgeball"] = function(atk,vic) atk:AddStat("dodgeballkill") end,
+        ["weapon_fists"] = function(atk) atk:AddStat("fistkill") end
+    }
+
+    local function TitlesPlayerDeath(vic,inf,atk)
```

Comment:
`hook.Add("PlayerDeath", "TitlesPlayerDeath", function(vic,inf,atk)    ...    end)`

Reply:
Think I got it.

Reply:
While I'll edit it so that the logic is in the hook.Add instead of a separate function, I will ask why the need for the change. I assume less code in the script?

Reply:
It is cleaner. The function logic is part of the same statement as the hook.Add. You don't need to name the function.

Reply:
Ah. This is more than fair.